coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -38,8 +38,9 @@ function MainNavigation() {
         <li>
           <MainNavigationLink
             icon={database}
-            to={`/index`}
+            to={Routes.DASHBOARD}
```

Comment:
This link should depend on the feature flag (if no FF than go to original Klaw etc), and should also have a test case.

Reply:
I thin hte test case is slighty overkill, as `useFeatureFlag` is already tested, but I added it anyway ^^ https://github.com/Aiven-Open/klaw/pull/2169/commits/4ba1b52e9cc4e445317b689813369c42973dfa18

Reply:
THANK YOU :D 

You know how it goes in my brain 🙈 

<img width="489" alt="Screenshot 2024-01-09 at 07 52 05" src="https://github.com/Aiven-Open/klaw/assets/943800/06cc6509-f0af-47f7-abd4-6e916ea697e8">